packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/DiagramRenderer.types.ts

```diff
@@ -21,6 +22,7 @@ export type FitViewLifecycle = 'neverRendered' | 'shouldFitview' | 'viewfit';
 
 export interface DiagramRendererProps {
   diagramRefreshedEventPayload: GQLDiagramRefreshedEventPayload;
+  diagramDescription: GQLDiagramDescription | undefined;
```

Comment:
Can this actuall be `undefined`? It looks like in DiagramRepresentation we only invoke the renderer once we have received the `diagramDescription` (and otherwise render an empty div).

Reply:
Done